pycode/README.rst

```diff
@@ -1,110 +1,36 @@
-      * epidemiology/epidata/defaultDict.py
-      * getcommutermigration
+* ``memilio-simulation``: Contains a python/pybind11 wrapper to the MEmilio C++ library. See `simulation README <memilio-simulation/README.md>`_.
 
-For all executables there are several run options.
-Use -h for more information or the `epidata-readme <epidemiology/epidata/README.rst>`_ in the corresponding subdirectory.
+See the corresponding directory for detailed installation and usage instruction.
 
-* ``epidemiology.secir``: Contains a python/pybind11 wrapper to the C++ code. An example is provided
-
-More details can be found in the corresponding subdirectories.
-
-Environment
------------
-
-We recommend to use a virtual environment.
-Therefore, do the following.
-
-Create virtiual environment with name "epi_venv" [name can be changed as you want] somewhere into the epidemiology folder(!) 
-
-.. code:: sh
-
-   # Setup virtual environment
-
-   python -m venv epi_venv
-   source epi_venv/bin/activate
-   pip install --upgrade pip
-
-If the virtual env is not created in the epidemiology-folder the default folder for writing and reading data files of the pidata package is in the site-package folder of the env, see output while writing the data. To avoid this use the -o flag, for details see `epidata-readme <epidemiology/epidata/README.rst>`_  or the `Documentation <https://hpc-against-corona.pages.gitlab.dlr.de/epidemiology/master/documentation/index.html>`_.
-
-
-Installation
-------------
-
-For developement of code use
+We recommend to use a virtual python environment to avoid dependency conflicts with other installed packages. On Linux, use the following commands to create and activate a virtual environment in directory called `epi_venv`:
 
 .. code:: sh
 
-    python setup.py develop
+    python -m venv epi_venv
+    source epi_venv/bin/activate
+    pip install --upgrade pip
 
+Refer to the `Python documentation <https://docs.python.org/3/library/venv.html>`_ for more information about virtual environments.
 
-To install, just enter
+Each package provides its own `setup.py`, so they can be installed seperately using the following commands
 
 .. code:: sh
 
+    cd <package directory>
     python setup.py install
 
-This builds the C++ extension and copies everything required to your site-packages. 
-
-All the requirements of the `C++ library <../cpp/README.md>`_ must be met in order to build and use the python bindings.
-CMake is executed internally by the `setup.py` script.
-
-Testing and Coverage
---------------------
-
-The following packages have to be installed to run the tests:
-
-- pyfakefs (creates fake directory to test that expected folders are created and data is written)
-- freezegun (freezes the time. Thus, the call today can be changed to a specific date.)
-
-To run the tests make 
+If you are a developer, use `python setup.py develop` instead when you are working on the code. Each package contains a test suite that can be run with
```

Comment:
You have quotes ('') here but in the Readmea it is shown as normal text. There should be some highlighting.

Reply:
I mixed up `rst` and `md` format there. Would be easier if all READMEs were the same format.